centaur/src/main/resources/standardTestCases/papi_v2_gcsa.test

```diff
@@ -0,0 +1,17 @@
+name: papi_v2_gcsa
+testFormat: WorkflowSuccess
+backends: [papi-v2-gcsa]
+
+files {
+  workflow: papi_v2_gcsa/papi_v2_gcsa.wdl
+  options-dir: ../../../../../target/ci/resources/
+  options-dir: ${?CROMWELL_BUILD_RESOURCES_DIRECTORY}
+  options: ${files.options-dir}/papi_v2_gcsa.options.json
+}
+
+metadata {
+  workflowName: papi_v2_gcsa
+  status: Succeeded
+  "outputs.papi_v2_gcsa.email": "cromwell@broad-dsde-cromwell-dev.iam.gserviceaccount.com"
+  "outputs.papi_v2_gcsa.scope": "https://www.googleapis.com/auth/monitoring.write https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/bigquery https://www.googleapis.com/auth/cloudkms https://www.googleapis.com/auth/devstorage.full_control https://www.googleapis.com/auth/compute https://www.googleapis.com/auth/genomics"
```

Comment:
It occurs to me that this ordering may not be stable, leading to inadvertent failures

That said I don't think WDL has any programming constructs that would help (e.g. sets, sorting, or a "contains" type of function)

Reply:
Good eye. I was hoping no one would notice, but since you brought it up...

I have no idea if this is even a stable API. Validating tokens using `tokeninfo` [used to be documented](https://web.archive.org/web/20150906185816/https://developers.google.com/identity/protocols/OAuth2UserAgent#tokeninfo-validation), even was [updated to a `/v3/`](https://web.archive.org/web/20160212040447/https://developers.google.com/identity/protocols/OAuth2UserAgent#tokeninfo-validation), but [validation has since disappeared](https://web.archive.org/web/20181108055947/https://developers.google.com/identity/protocols/OAuth2UserAgent). For all I know this endpoint is destined for the google graveyard like so many other products before. If someone does know of a better (and documented?) way to [test tokens](https://developers.google.com/apis-explorer/#search/tokeninfo/m/oauth2/v2/oauth2.tokeninfo) please let me know.

In the meantime I'll add `jq` to the `command` and then use that API to sort the space separated array. 🤞 that the whole REST API sticks around and doesn't break our centaur test.